components/Cards.js

```diff
@@ -20,3 +20,47 @@
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+import axios from 'axios'
+
+axios.get('https://lambda-times-api.herokuapp.com/articles')
+.then(e => {
+    function articleMaker(obj){
```

Comment:
would recommend that you pull this function definition outside of the .then() clause
generally we avoid defining long functions inside of other functions